pkg/api/plugin.go

```diff
@@ -23,7 +23,7 @@ const (
 	PluginStepInitialize                          PluginStep = "Initialize"
 	PluginStepInitializeUpdateBlob                PluginStep = "InitializeUpdateBlob"
 	PluginStepResetUpdateBlob                     PluginStep = "ResetUpdateBlob"
-	PluginStepBlobDoesNotExist                    PluginStep = "BlobDoesNotExist"
+	PluginStepBlobEtcdBlobExists                  PluginStep = "BlobDoesNotExist"
```

Comment:
you are not changing the value, so the value and the step say 2 different things

Reply:
fail by me - too much hurry.  Thanks.